ansible/roles/kong-api/defaults/main.yml

```diff
@@ -122,6 +122,7 @@ uci_gql_service_url: "http://gql-service:8080"
 uci_registry_service_url: "http://fusionauth-service:9011"
 uci_admin_service_url: "http://uci_api-service:9999"
 notification_service_url: "http://notification-service:9000"
+program_service_url: "http://program-service:6000"
```

Comment:
@Hari-1027 We dont have any service named `program-service` in Sunbird. Looks like this PR should be for vidyadaan. Please connect with KartheekP to clarify.

Reply:
@keshavprasadms Sorry for the inconvenience. Checked with @pallakartheekreddy and corrected the branch. A new PR has been raised. we are closing this one.